java/README.md

```diff
@@ -14,3 +14,56 @@
 - Depth First Search
 - Breadth First Search
 - Binary Search
+
+### Data Structures:
+- Stack
+  - push(x): Push element x onto stack.  
+  - pop(): Removes the element on top of the stack.  
+  - top(): Get the top element.  
+  - empty(): Return whether the stack is empty.  
```

Comment:
`Push`, `Get`, and `Return` are inconsistent with comments for other operations (`Removes`, `Inserts`, `Deletes`, `Returns`) as these are in the Imperative Mood (commands), while others are descriptive.